shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/metrics/MetricsServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package com.redhat.service.bridge.shard.operator.metrics;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+
+@ApplicationScoped
+public class MetricsServiceImpl implements MetricsService {
+
+    private static final String MANAGER_REQUEST_METRICS = "http.manager.request";
+
+    @Inject
+    MeterRegistry meterRegistry;
+
+    @ConfigProperty(name = "event-bridge.sso.grant-options.password.user-id")
+    String shardId;
+
+    @Override
+    public void updateManagerRequestMetrics(ManagerRequestType requestType, ManagerRequestStatus status) {
```

Comment:
@vaibhavjainwiz Is it also useful to include the HTTP status received when trying to call the Manager here as a tag?